### PR TITLE
add input documentation

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Postman] Create Input documentation](https://trello.com/c/GadvpI3c/640-postman-create-input-documentation)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -5298,6 +5298,173 @@
...
+												},
+												{
+													"key": "name",
+													"value": "FERT",
+													"description": "Text",
+													"disabled": true
+												},
+												{
+													"key": "crops",
+													"value": "[\"corn\"]",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:5380]: Can we keep this value as a single input in the example? Just to make the call like `?crops=corn'

**Review by fabyan-alexander — APPROVED** [review]
Nice work!
Just a single suggestion :smiley:

**Review by Vinniekun — APPROVED** [review]

**Comment by allanpetersoon** [comment]
This PR should only be approved when we insert in the official documentation the availability of the endpoints of varieties

**Comment by allanpetersoon** [comment]
This PR should only be merged when the varieties doc becomes public

**Pull request closed without merging**